src/views/AuthView/AuthView.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from "react";
+import Login from "../../components/Login";
+import Register from "../../components/Register";
+import styles from "./AuthView.module.scss";
+
+function AuthView(props) {
+  const [isLogin, setIsLogin] = useState(true);
+
```

Comment:
Like mentioned in the App.js comment, might need to bring back the useEffect unless you have something else in mind.

Reply:
and then here would be

```
  useEffect(() => {
    props.setRetailerLoggedIn(isLogin);
  }, [isLogin, props])
```